php.ini-development

```diff
@@ -517,6 +517,15 @@ report_memleaks = On
 ; This setting is on by default.
 ;report_zend_debug = 0
 
+; Set this to disable non-ASCII logging (more secure).
```

Comment:
The more secure comments is very misleading and it just show on broken loggers that can't handle UTF-8 for example which is a minority. It should be really clear that it is a work around for broken loggers.

Btw. there is no security issue in PHP. This is just a problem of the loggers if there are some security issues as RFC does not prohibit anything.